spec/models/generic_work_spec.rb

```diff
@@ -14,6 +14,16 @@
     it { is_expected.to eq 'curation_concerns_generic_work' }
   end
 
+  describe '#object_state' do
+    let(:work) { described_class.new }
+    let(:inactive) { ::RDF::URI('http://fedora.info/definitions/1/0/access/ObjState#inactive') }
```

Comment:
I doubt that this is going to get published in `rdf-vocab` as it's not a widely used vocabulary.